.github/pull_request_template.md

```diff
@@ -26,6 +26,5 @@ dispatcher where…", "Improve our handling of…", etc.
 
 ## Tests and Linting
 - [ ] This branch is up-to-date with the `develop` branch.
-- [ ] Ran `make create_db_diagram` command.
```

Comment:
Since we have an Action for this, I don't think we need this line anymore. I could see the benefit of having it more generally tho, so I'm open to putting it back.